com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/cf/detect/MtaMetadataEntityAggregator.java

```diff
@@ -0,0 +1,23 @@
+package com.sap.cloud.lm.sl.cf.core.cf.detect;
+
+import com.sap.cloud.lm.sl.cf.core.cf.detect.entity.MetadataEntity;
+import com.sap.cloud.lm.sl.cf.core.model.DeployedMta;
+import com.sap.cloud.lm.sl.common.util.JsonUtil;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+@Component
+public class MtaMetadataEntityAggregator {
+
+    @Autowired
+    private MtaMetadataExtractorFactory<MetadataEntity> metadataExtractorFactory;
+
+    public DeployedMta aggregate(List<MetadataEntity> entities) {
```

Comment:
We talked - Each extractor produces DeployedMtaModule/DeployedMtaResource(Service) and here in this class return new DeployMta(<list-of-deployed-modules>, <list-of-deployed-resources>)